Sources/IO/Misc/PDBReader/example/index.js

```diff
@@ -26,6 +26,7 @@ const sphereActor = vtkActor.newInstance();
 const stickActor = vtkActor.newInstance();
 
 filter.setInputConnection(reader.getOutputPort());
+filter.setHideElement('H');
```

Comment:
it does expect an array of string which will be compared to the elements.json file to hide them